docs/src/main/sphinx/connector/hive-security.rst

```diff
@@ -519,6 +519,31 @@ These rules govern the privileges granted on specific tables.
 
 * ``filter_environment`` (optional): environment use during filter evaluation.
 
+.. note::
```

Comment:
Do now use note for such a long section please

Reply:
sorry, Do `not` not `now`, right?

Reply:
yes .. sorry for the typo